int64 PrinciplesOfRobotics
int64 HardwareAbstraction
int64 Safety 
int64 Ros
int64 Vision
---
string Answer
